spec.html

```diff
@@ -32006,7 +32006,7 @@
           1. Let _O_ be the *this* value.
           1. Perform ? ValidateTypedArray(_O_).
           1. Let _value_ be ? ToNumber(_value_).
-          1. Let _len_ be ? _O_.[[ArrayLength]].
+          1. Let _len_ be _O_.[[ArrayLength]].
```

Comment:
> ValidateAtomicAccess is the other case in the specs we can verify [[ArrayLength]] being accessed after an observable step where we do not check for a detached buffer.

ValidateAtomicAccess is always preceded by ValidateSharedIntegerTypedArray, so the buffer can never be detached.